packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1057,7 +1056,7 @@ function commitUnmount(
   finishedRoot: FiberRoot,
   current: Fiber,
   nearestMountedAncestor: Fiber,
-  renderPriorityLevel: ReactPriorityLevel,
+  renderPriorityLevel: LanePriority,
```

Comment:
Switched all these methods to pass around a LanePriority now. After this, the only references to this type are in ReactFiberLanes (for converting between), SchedulerWithReactIntegration, and the DevTools.